src/haz3lschool/SyntaxTest.re

```diff
@@ -7,6 +7,10 @@ type syntax_result = {
   percentage: float,
 };
 
+/*
+ Finds whether a variable with the same name as
```

Comment:
You can just say "the variable `name`" rather than "a variable with the same name as the argument "name"" here and similar elsewhere.